plugins/report-portal/src/components/ReportPortalGlobalPage/ReportPortalGlobalPage.tsx

```diff
@@ -13,14 +14,38 @@ export type ReportPortalGlobalPageProps = {
 };
 
 export const ReportPortalGlobalPage = (props: ReportPortalGlobalPageProps) => {
+  const config = useApi(configApiRef);
+  const sendAddress = config.getString('reportPortal.supportEmail');
   return (
     <PageWithHeader
       themeId={props.theme ?? 'app'}
       subtitle={props.subtitle ?? 'View all report portals'}
       title={props.title ?? 'Report Portal'}
     >
       <Content>
-        <Grid container wrap="nowrap">
+        <Grid container>
+          <Grid
+            xs={12}
+            item
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'right',
+            }}
+          >
+            Don't see your instance here? &nbsp;
+            <Button
+              style={{ textTransform: 'none', padding: '5px 10px' }}
+              color="primary"
+              variant="outlined"
+              target="_blank"
+              href={`https://mail.google.com/mail/?view=cm&fs=1&tf=1&to=${encodeURIComponent(
+                sendAddress,
+              )}&su=Request%20to%20add%20a%20report%20portal%20instance&body=Requesting%20to%20add%20a%20new%20Report%20Portal%20instance%20to%20Red%20Hat%20Experience%20Platform.%0A%0AHostname%3A%0AProject%20Manager%3A%0AInstance%20Admin%20email%3A`}
```

Comment:
@yashoswalyo I think the email subject and body should be stored in a string variable first.

```ts
const body = encodeURIComponent(`<multiline
email body>
`);
```